lib/Dialect/TritonGPU/Transforms/Combine.cpp

```diff
@@ -575,16 +576,68 @@ static int computeCapabilityToMMAVersion(int computeCapability) {
 static SmallVector<int64_t, 2>
 mmaVersionToShapePerWarp(int version, const ArrayRef<int64_t> &shape,
```

Comment:
`const ArrayRef<int64_t> &shape`

Should you also change this to `const ArrayRef<int64_t> shape`?